Dockerfile

```diff
@@ -81,9 +81,6 @@ RUN --mount=type=cache,target=${DEST_FOLDER}/yarn-cache,uid=1000,gid=1000 yarn
 # build all packages in packages/**
 RUN yarn build
 
-# Build locksmith and other apps separately to reduce startup time on heroku
-RUN yarn apps:build
-
```

Comment:
Don't need to build it separately.

Reply:
Hum, will that create more timeouts on Heroku?